traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -89,17 +90,28 @@ import (
 )
 
 // Authenticated ...
-var Authenticated = true
+const Authenticated = true
 
 // NoAuth ...
-var NoAuth = false
+const NoAuth = false
+
+const perlBypass = true
```

Comment:
Would you mind adding comments explaining what these variables are doing?

Reading the routes below, it wasn't obvious to me what "perl bypass" meant, I had to go look it up on the mailing list and make some inferences.